ci/tests/broker_db_lifecycle

```diff
@@ -20,9 +20,9 @@ appId="$(uuidgen)"
 bindIp="${bindIp:-10.244.2.16}"
 
 cl="$(curl ${1:-} -s -X GET -H "X-Broker-API-Version: 2.4" -H "Content-Type: application/json" "http://cfadmin:cfadmin@${bindIp}:8888/v2/catalog")"
-planId="$(echo $cl | jq '.services[0].plans[1].id' | tr -d '"')"
-planName="$(echo $cl | jq '.services[0].plans[1].name' | tr -d '"')"
-serviceId="$(echo $cl | jq '.services[0].id '| tr -d '"')"
+planId="$(echo $cl | jq -r '.services[0].plans[1].id')"
```

Comment:
The variable name `cl` is non-descriptive to what it is, `catalog` might be better or perhaps `catalogJSON`.

These lines should be using the BASH HERESTRING instead of `echo ${variable}` eg.:
```sh
planId="$(jq -r '.services[0].plans[1].id' <<< "${cl}")" 
```